react/example/app.js

```diff
@@ -1,23 +1,34 @@
-import React from 'react'
-import MaskedInput from '../src/reactTextMask'
+import React from 'react';
+import MaskedInput from '../src/reactTextMask';
 
-export default React.createClass({
-  render() {
-    return (
-      <form className='form-horizontal'>
-        <div className='form-group'>
-          <label htmlFor='1' className='col-sm-2 control-label'>Masked input</label>
+export default () => (
```

Comment:
The change is needed to get rid of createClass which is deprecated. Functional components have been around since react 14. This should all be fully backwards compatible. I can run the tests with React 14 and see if you'd like?